python/lsst/analysis/tools/tasks/calexpSummaryAnalysis.py

```diff
@@ -0,0 +1,69 @@
+# This file is part of analysis_tools.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+from __future__ import annotations
+
+__all__ = (
+    "CalexpSummaryAnalysisConfig",
+    "CalexpSummaryAnalysisTask",
+)
+
+from lsst.pipe.base import InputQuantizedConnection, OutputQuantizedConnection, QuantumContext
+from lsst.pipe.base import connectionTypes as cT
+
+from ..interfaces import AnalysisBaseConfig, AnalysisBaseConnections, AnalysisPipelineTask
+
+
+class CalexpSummaryAnalysisConnections(
+    AnalysisBaseConnections,
+    dimensions=("visit", "band", "detector"),
+    defaultTemplates={"inputName": "calexp", "outputName": "calexpSummary"},
+):
```

Comment:
I am not sure this is what you want. In the calibrate task I think you can just write out the `ExposureSummaryStats` in a connection, and then just read that in here, that would be way faster and cheaper than loading the whole exposure. There is already a Storage Class type for it in the butler: https://github.com/lsst/daf_butler/blob/main/python/lsst/daf/butler/configs/storageClasses.yaml#L33C3-L33C23 . This will need a change on the pipe_base side of the ticket.

Reply:
BUT if we don't want to write the summary stats out for some good reason, this here looks fine.

Reply:
I am fine for time reasons if you want to skip this for this ticket

Reply:
I can't see where in calibrate.py `ExposureSummaryStats` are being written, so while I agree it would be much cheaper to just load the stats, it doesn't seem to be possible at thiss stage. Maybe I've missed something, though.

Reply:
Actually, is there a reason to load it like this at all for this? It make sense if you wanted to stream them out as you go, but if you are all the way in analysis tools running a pipeline for follow up, why would you not just get it from the summary table that has the whole visit in it?

Reply:
I agree that one may want to just do as you suggest. However, I'm looking at it as someone who isn't very familiar with analysis_tools - if calexpSimmaryAnalysis is retargeted in the yaml file, then it's immediately clear to someone that they can use that analysis tool task to replicate the metrics.

Reply:
Can you try changing your connection to `calexp.summaryStats` talking to Eli I think this is not possible with component disassembly.

Reply:
calexp.summaryStats works, so I can change it to that (if that's what you mean).

Reply:
I've changed it to `calexp.summaryStats`.